backup_and_restore/application_backup_and_restore/installing/installing-oadp-mcg.adoc

```diff
@@ -12,7 +12,11 @@ toc::[]
 
 You install the Openshift API for Data Protection (OADP) with Multicloud Object Gateway (MCG) by installing the OADP Operator. The Operator installs link:https://{velero-domain}/docs/v1.7/[Velero 1.7].
 
+<<<<<<< HEAD
 You configure xref:../../../backup_and_restore/application_backup_and_restore/installing/installing-oadp-mcg.adoc#installing-oadp-mcg[Multicloud Object Gateway] as a backup location.
+=======
+MCG is a component of {rh-storage}. You configure MCG as a backup location in the `DataProtectionApplication` custom resource (CR).
+>>>>>>> Change 'OpenShift Container Storage' to 'OpenShift Data Foundation'
```

Comment:
You've got some merge conflict resolution metadata that slipped in here (lines 15-17 & 19, I think). Not sure if this will resolve the build errors but it's possible. 🤞